addons/stock_account/models/product.py

```diff
@@ -757,23 +765,21 @@ class ProductCategory(models.Model):
         'account.account', 'Stock Valuation Account', company_dependent=True,
         domain="[('company_id', '=', allowed_company_ids[0]), ('deprecated', '=', False)]", check_company=True,
         help="""When automated inventory valuation is enabled on a product, this account will hold the current value of the products.""",)
+    property_stock_account_production_cost_id = fields.Many2one(
+        'account.account', 'Production Account', company_dependent=True,
+        domain="[('company_id', '=', allowed_company_ids[0]), ('deprecated', '=', False)]", check_company=True,
+        help="""This account will be used as a valuation counterpart for both components and final products for manufacturing orders.
+                If there are any workcenter/employee costs, this value will remain on the account once the production is completed.""")
 
-    @api.constrains('property_valuation', 'property_stock_valuation_account_id', 'property_stock_account_output_categ_id', 'property_stock_account_input_categ_id')
-    def _check_valuation_accouts(self):
+    @api.constrains(*STOCK_PROPERTY_FIELDS, 'property_valuation')
```

Comment:
```suggestion
    @api.constrains(lambda self: self._get_account_property_fields() + ['property_valuation'])
```
Following my previous comment, if you set a getter method for that list, the `constrains` will be properly updated once `mrp` is installed.